modules/exploits/windows/http/desktopcentral_deserialization.rb

```diff
@@ -26,52 +26,51 @@ def initialize(info = {})
         the complete fix for the remote code execution vulnerability is now
         available in build 10.0.479."
       },
-      'Author'           => [
+      'Author'          => [
         'mr_me', # Discovery and exploit
         'wvu'    # Module
       ],
-      'References'       => [
+      'References'      => [
         ['CVE', '2020-10189'],
         ['URL', 'https://srcincite.io/advisories/src-2020-0011/'],
         ['URL', 'https://srcincite.io/pocs/src-2020-0011.py.txt'],
         ['URL', 'https://twitter.com/steventseeley/status/1235635108498948096'],
         ['URL', 'https://www.manageengine.com/products/desktop-central/remote-code-execution-vulnerability.html']
       ],
-      'DisclosureDate'   => '2020-03-05', # 0day release
-      'License'          => MSF_LICENSE,
-      'Platform'         => 'windows',
-      'Arch'             => [ARCH_CMD, ARCH_X86, ARCH_X64],
-      'Privileged'       => true,
-      'Targets'          => [
+      'DisclosureDate'  => '2020-03-05', # 0day release
+      'License'         => MSF_LICENSE,
+      'Platform'        => 'win',
```

Comment:
Huh, apparently this is a thing and I was not aware of it at all, but looks good. Approved.

Reply:
Either will work. Specifically, it is `win`, and I use that in other modules. I hate it. I might change it later, haha. But it's consistent now.